src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -112,25 +116,30 @@ public static Calendar now(double timeZone) {
     }
 
     /**
-     * Convert a date string and time string into a Date object. Returns null on error.
-     *
-     * @param inputDate
+     * Convert a date string and time string of only the hour into a Date object.
+     * If the hour is 24, it is converted to 23:59. Returns null on error.
+     *  @param inputDate
```

Comment:
revert this change.

Reply:
I am not changing the behavior of this function. I am simply adding the undocumented behavior to the javadoc

Reply:
```diff
+ *  @param inputDate
```
I was talking about this extra space that you gave before `@param`

Reply:
I see 😅, missed out that small space. Thank you for pointing it out.